gcassert.go

```diff
@@ -134,8 +139,8 @@ func (v assertVisitor) Visit(node ast.Node) (w ast.Visitor) {
 					}
 				}
 				lineInfo.directives = append(lineInfo.directives, directive)
+				v.directiveMap[lineNumber] = lineInfo
```

Comment:
This change is needed due to `continue COMMENT_LIST` above, right? It seems like previously we could silently ignore some directives if we found `inline` directive - should we add a test for that?

Reply:
I moved it to avoid writing an empty `lineInfo` if all directives were invalid.

The `continue COMMENT_LIST` feels very weird to me, but I guess it doesn't matter because `inline` is the only valid directive for a func declaration anyway.

Reply:
I changed it to a simple `continue` and added a test that we don't ignore a directive after a func-level `inline`.

Reply:
Thanks!